app/src/request-templates.ts

```diff
@@ -0,0 +1,151 @@
+import { PredicateType } from '@aries-framework/core'
+import { ProofRequestTemplate, ProofRequestType } from 'aries-bifold'
+
+export const proofRequestTemplates: Array<ProofRequestTemplate> = [
+  {
+    id: 'BC:5:FullName:0.0.1:indy',
+    name: 'Full name',
+    description: 'Verify the full name of a person',
+    version: '0.0.1',
+    payload: {
+      type: ProofRequestType.Indy,
+      data: [
+        {
+          schema: 'XUxBrVSALWHLeycAUhrNr9:2:Person:1.0',
+          requestedAttributes: [
+            {
+              name: 'given_names',
+              restrictions: [{ schema_id: 'XUxBrVSALWHLeycAUhrNr9:2:Person:1.0' }],
```

Comment:
We need to make all the restrictions by credential definition id, not schema id. Anyone can create a credential with a given schema_id, so anyone can issue themselves a credential that passes this proof request.

Reply:
going to approve this to get it in the codebase, but I'll create a separate issue for changing the restrictions